wrangler-docs/functions/date-and-time-functions.md

```diff
@@ -0,0 +1,1010 @@
+```
+
+## GetTimestamp
+Returns the `timestamp` represented by the given input.
+
+#### Namespace
+`datetime`
+
+#### Input
+timestamp(`String`)
+
+#### Output
+timestamp (`LocalDateTime`)
+
+#### Example
+Use this function to convert the String to a timestamp.
+```
+set-column date datetime:GetDate('2008-08-18 22:30:52')
+```
+
+## CurrentDate
+Returns the `date` when the function is invoked.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+date (`LocalDate`)
+
+#### Example
+Use this function to add the current date to a column.
+```
+set-column date datetime:CurrentDate()
+```
+
+## CurrentTime
+Returns `time` when the function in invoked.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+time (`LocalTime`)
+
+#### Example
+Use this function to add the current time to a column.
+```
+set-column time datetime:CurrentTime()
+```
+
+## CurrentTimeMS
+Returns `time` in milliseconds when the function in invoked.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+time (`long`)
+
+#### Example
+Use this function to add the current time to a column.
+```
+set-column timems datetime:CurrentTimeMS()
+```
+
+## CurrentTimestamp
+Returns `timestamp` that include `date` and `time` when the function is invoked.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+timestamp (`LocalDateTime`)
+
+#### Example
+Use this function to add the current timestamp to a column.
+```
+set-column timestamp datetime:CurrentTimestamp()
+```
+
+## CurrentTimestampMS
+Returns `timestamp` that include `date` and `time`. The `time` component includes milli seconds.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+timestamp (`long`)
+
+#### Example
+Use this function to add the current timestamp including milli seconds to a column.
+```
+set-column timestampms datetime:CurrentTimestampMS()
+```
+
+## CurrentTimestampMicro
+Returns `timestamp` that include `date` and `time`. The `time` component includes micro seconds.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+timestamp (`long`)
+
+#### Example
+Use this function to add the current timestamp including micro seconds to a column.
+```
+set-column timestampmicro datetime:CurrentTimestampMicro()
+```
+
+## CurrentTimestampNano
+Returns `timestamp` that include `date` and `time`. The `time` component includes nano seconds.
+
+#### Namespace
+`datetime`
+
+#### Input
+None
+
+#### Output
+timestamp (`long`)
+
+#### Example
+Use this function to add the current timestamp including nanoseconds to a column.
+```
+set-column timestampnano datetime:CurrentTimestampNano()
+```
+
+## DateFromDaysSince
+Returns a date object by adding an integer to a baseline date.
+The integer can be negative to return a date that is earlier than the baseline date.
+If baseline is null, null will be returned. If the number is null, it will be treated as 0. 
+
+#### Namespace
+`datetime`
+
+#### Input
+number (`Integer`), \[baseline date(`LocalDate`)\]
```

Comment:
baseline date shouldn't be in brackets should it? From the description it seems like it's a required argument. If it's optional the description should be updated.

Reply:
This is because DateFromDaysSince has two methods, one of them does not take the LocalDate, I have updated the description to match the behavior